contracts/IRegistry.sol

```diff
@@ -26,7 +26,7 @@ interface IRegistry {
       * @param orgMetadataURI  MetadataURI of organization to create, must be unique registry-wide.
       * @param members  Array of member addresses to seed the organization with.
       */
-    function createOrganization(bytes32 orgId, bytes orgMetadataURI, address[] members) external;
+    function createOrganization(bytes32 orgId, bytes calldata orgMetadataURI, address[] calldata members) external;
```

Comment:
Is calldata cheaper then memory?

Reply:
calldata is similar to memory.  For all the external methods it is mandatory to keep it as calldata than memory.